package.json

```diff
@@ -12,6 +12,11 @@
   "publishConfig": {
     "provenance": true
   },
+  "keywords": [
+    "web-components",
```

Comment:
An odd order. But it matches that of GitHub, whose order doesn't seem changeable.

<img width="337" alt="image" src="https://github.com/user-attachments/assets/9dcc3689-fb9d-4fe8-b582-6835408ead14" />